rs080n_moveit_config/CMakeLists.txt

```diff
@@ -8,4 +8,3 @@ catkin_package()
 install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
   PATTERN "setup_assistant.launch" EXCLUDE)
 install(DIRECTORY config DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
-install(DIRECTORY script DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
This seems like an unrelated change (a necessary one, but unrelated to the linking problem).